### DISCUSSION TITLE
(linux/arm64/v8)

### DISCUSSION THREAD

**Discussion by HexaNLD** [Hubro/clipface#67] [Q&A] [answered]
Whenever I run the docker script I get this error: "WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested".

**Comment by Hubro** [comment]
Which script do you mean?

Unfortunately I don't own an arm64 device so I haven't been able to provide a pre-built image for that platform. The official Docker image is currently only built for amd64.

GitHub Workflow also doesn't support arm64, so I haven't been able to go that route either.

I am looking into maybe using the free tier of CircleCI to build images for multiple platforms.

**Comment by HexaNLD** [comment]
I don't know why I said script, but I mean this part:
"docker run -d \
  --name clipface \
  -v /host/path/to/clips:/clips \
  -p 80:80 \
  tomsan/clipface:latest"

Also is there any way to build the image yourself? I'm pretty new to docker so I don't know if that is possible and if possible how to do it.

**Comment by Hubro** [comment, marked as answer]
@ThijmenBlokland It's possible, yes. Check out the `build-docker-image.sh` script in the root of the project :)

I should probably add a developer section to the README at some point.
